.aws/src/SqsLambda.ts

```diff
@@ -0,0 +1,76 @@
+import { Resource } from 'cdktf';
+import { Construct } from 'constructs';
+import { config } from './config';
+import { PocketVPC } from '@pocket-tools/terraform-modules';
+import { PocketSQSWithLambdaTarget } from '@pocket-tools/terraform-modules';
+import { LAMBDA_RUNTIMES } from '@pocket-tools/terraform-modules';
+import { ssm } from '@cdktf/provider-aws';
+import { PocketPagerDuty } from '@pocket-tools/terraform-modules';
+
+export class SqsLambda extends Resource {
+  constructor(
+    scope: Construct,
+    private name: string,
+    vpc: PocketVPC,
+    pagerDuty?: PocketPagerDuty
+  ) {
+    super(scope, name);
+
+    const { sentryDsn, gitSha } = this.getEnvVariableValues();
+
+    new PocketSQSWithLambdaTarget(this, 'sqs-event-consumer', {
+      name: `${config.prefix}-Sqs-Event-Consumer`,
+      batchSize: 10,
+      batchWindow: 60,
+      sqsQueue: {
+        maxReceiveCount: 3,
+        visibilityTimeoutSeconds: 300,
+      },
+      lambda: {
+        runtime: LAMBDA_RUNTIMES.NODEJS14,
+        handler: 'index.handler',
```

Comment:
This is more of a learning/clarifying question - we do have this here and we have the file `../../lambda/index.ts` which is exporting the function `handler`. How does that function get mapped / "picked up" by the lambda? I don't see a explicit path pointing to that file/function 🤔

Reply:
responded in slack!